candi/bashible/common-steps/node-group/099_reboot.sh.tpl

```diff
@@ -37,7 +37,8 @@ kill -KILL $(ps ax | grep containerd-shim | grep -v grep |awk '{print $1}')
 exit 0
 {{- end }}
 
-bb-deckhouse-get-disruptive-update-approval
+bb-deckhouse-get-disruptive-update-approval {{ .nodeGroup.disruptions.approvalMode | default "Manual" | quote }}
```

Comment:
I think we've got logical misconfiguration. If we use rolling update we should drop the node and order the new one. But we get disruptive approval at the end of bashible scripts. I think we should add additional approval at the bashible start if rolling update is required to prevent time loss on bashible execution.